apps/design/shared/src/document_svg.tsx

```diff
@@ -49,8 +50,9 @@ export function SvgTextBox({
         <text
           // eslint-disable-next-line react/no-array-index-key
           key={textLine + index}
-          x={0}
+          x={align === 'right' ? width : 0}
           y={(index + 1) * lineHeight}
+          textAnchor={align === 'left' ? 'start' : 'end'}
```

Comment:
[textAnchor](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/text-anchor#example) is a crazy property. It moves the text to a diff position in relation to the `x` coordinate (rather than just moving the text within the content box like we're used to with HTML). So if you switch `textAnchor` to `'end'`, you need to move the `x` coordinate to the other end of the content box to get the box in the same position as it was before. I encapsulated this adjustment within the rendering so that the `TextBox` component behaves more like what we're used to.